From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Liav A <liavalb@gmail.com>
Date: Sat, 12 Aug 2023 19:15:27 +0300
Subject: [PATCH] Don't redeclare signbit function

We have it already implemented so simply use what we have.
---
 lib/math.in.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/math.in.h b/lib/math.in.h
index 2b595710530c99ec4f4a6681e4e2b623716028d1..419ab4a609ce0ecd753bb9673819144d81836d8b 100644
--- a/lib/math.in.h
+++ b/lib/math.in.h
@@ -2344,7 +2344,7 @@ _GL_WARN_REAL_FLOATING_DECL (isnan);
 #endif
 
 
-#if @GNULIB_SIGNBIT@
+#if 0
 # if (@REPLACE_SIGNBIT_USING_GCC@ \
       && (!defined __cplusplus || __cplusplus < 201103))
 #  undef signbit
